content/guides/ai-studio/index.md

```diff
@@ -0,0 +1,46 @@
+---
+rank: 0
+related_guides:
+	- box-ai/ai-studio/create-agents
+	- box-ai/ai-tutorials/ask-questions
+	- box-ai/ai-tutorials/generate-text
+---
+
+# Box AI Studio
+
+<Messsage type='caution'>
+Box AI Studio is available only for Enterprise Advance accounts.
+</Message>
+
+[Box AI Studio][ai-studio] allows you to build and manage custom AI agents to
+best suit your business needs. For example, you can create an AI agent
+that acts as a compliance consultant, answering questions regarding customer
+documentation with the FedRAMP Moderate compliance in mind. 
```

Comment:
Can we add information where those agents are later available for use, as this is not clear.